site/public/css/server.css

```diff
@@ -460,12 +460,11 @@ td>a.active-sort {
 }
 
 #config_search {
-    padding-right: 20px;
+    padding-right: 45px;
 }
 
 #clear_search_button {
-    margin-left: -20px;
-    padding-right: 20px;
+    padding-right: 6px;
     color: red;
 }
```

Comment:
Since these styling rules are specific to the page, they should be moved into ```admin-gradeable.css```. A lot of code was just dumped into ```server.js/css``` initially, so it's good to move it out to its respective file when you can.